backends/client.go

```diff
@@ -4,6 +4,7 @@ type Client interface {
 	Load() ([]byte, error)
 	Save([]byte) error
 	Backup() error
+	SafeLocation() string
```

Comment:
For what do you intent to use different backend clients anyway? A backend client to store the safe in a database, on a remote host (e.g. "ssh backend client") and so one?
Even then we will definitely need to show the safe location. Otherwise you won't be able to differentiate between two — say database storage locations.

Example:

Image pick is configured to use an ssh backend client and you want to sync it with a different host:

```sh
$ pick sync ssh://MY_HOST_2:/path/to/pick.safe
Enter your master password for safe 'ssh://MY_HOST_1:/path/to/pick.safe' (local)
..
Enter your master password for safe 'ssh://MY_HOST_2:/path/to/pick.safe' (remote)
..
Syncing 'test'.. done
```